skglm/datafits/group.py

```diff
@@ -50,6 +52,21 @@ def get_lipschitz(self, X, y):
 
         return lipschitz
 
+    def get_lipschitz_sparse(self, X_data, X_indptr, X_indices, y):
+        grp_ptr, grp_indices = self.grp_ptr, self.grp_indices
+        n_groups = len(grp_ptr) - 1
+        n_rows = len(y)
```

Comment:
since this variable is used only once, you can avoid defining it
[in addition, we prefer the sklearn convention `n_samples`]